apluslms-yamlidator/apluslms_yamlidator/utils/__init__.py

```diff
@@ -0,0 +1,12 @@
+BOOLEANS = {
+    0: False, 1: True,
+    '0': False, '1': True,
+    'false': False, 'true': True,
+    'False': False, 'True': True
+}
+
+def convert_to_boolean(val):
+    if val not in BOOLEANS:
```

Comment:
Technically this makes two queries to BOLEANS (relatively fast). I'm not sure if this alternative would be slower or faster..

```
try:
  return BOOLEANS[val]
except KeyError:
  raise ValueError(...)
```

Reply:
The time you gain from only calling val once isn't as big as the time you lose with the try except. Then again we can probably assume that most of the time an error won't be raised. But the method isn't going to be called that much (usually 0 or 1 times) so the difference isn't significant either way :p

Reply:
Yep. Thus I think the current solutions is totally fine. Wanted to leave this for the record and as something I would like to benchmark some day. Reference implementation in configparser does even call `key.lower()` twice..